cmd/afsPmf/afsPmf.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/exception"
+	"github.com/vertgenlab/gonomics/fileio"
+	"github.com/vertgenlab/gonomics/popgen"
+	"log"
+)
+
+func afsPmf(vcfFile string, outFile string, unPolarized bool) {
+	g, err := popgen.VcfToAfs(vcfFile, !unPolarized) //VcfToAfs is written in terms of polarized, so this is inverted here.
+	exception.FatalOnErr(err)
+	out := fileio.EasyCreate(outFile)
+
+	numAlleles := g.Sites[0].N
+	frequencySlice := make([]int, numAlleles)
+
+	for _, site := range g.Sites {
+		frequencySlice[site.I]++
+	}
+	_, err = fmt.Fprintf(out, "AlleleFrequency\tNumAlleles\n")
+	exception.PanicOnErr(err)
+	for i := 1; i < len(frequencySlice); i++ {
+		_, err = fmt.Fprintf(out, "%v\t%v\t%f\n", i, frequencySlice[i], float64(frequencySlice[i])/float64(len(g.Sites)))
```

Comment:
I was expecting %d instead of %v.  It looks like %v works well too.  Not sure if one is more clear than the other.